### PR TITLE
clue: add dragon scimitar or to falco the bard cluestep

### LINKED ISSUE #13031: Dragon scim (or) - falo clue master step
Dscim ornament version works for this step 
image

### PR DESCRIPTION
**Pull request by WolffTech**
Fixes #13031

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
 public class FaloTheBardClue extends ClueScroll implements TextClueScroll, NpcClueScroll
 {
 	private static final List<FaloTheBardClue> CLUES = ImmutableList.of(
-		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", item(DRAGON_SCIMITAR)),
+		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", item(DRAGON_SCIMITAR), item(DRAGON_SCIMITAR_OR)),
```
Comment (makingcomment) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java:53]: I think you need an `any`.

**Comment by Adam-** [comment]
tip: you can use `::clue A blood red weapon, a strong curved sword, found on the island of primate lords.` in game and it will make that clue active, so you can test changes like this.

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java (hunk 1 above)
Comment (WolffTech) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java]: I can't believe I missed this. Thank you for the help, I have made the proper changes and updated the commit.

**Comment by WolffTech** [comment]
I have updated the commit. I made the infobox text show `Any Dragon Scimitar`. If it should be something different please let me know and I'll change it. This is my first time using Git so if there's any issues please let me know. Thanks!

**Comment by Nightfirecat** [comment]
I'd just use the text `Dragon scimitar`, like it would be for a non-ornamented item.

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java [hunk 2]
```diff
@@ -50,7 +50,7 @@
 public class FaloTheBardClue extends ClueScroll implements TextClueScroll, NpcClueScroll
 {
 	private static final List<FaloTheBardClue> CLUES = ImmutableList.of(
-		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", item(DRAGON_SCIMITAR)),
+		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", any("Dragon Scimitar", item(DRAGON_SCIMITAR), item(DRAGON_SCIMITAR_OR))),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java:53]: (commenting to commit via GitHub's UI)
```suggestion
		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", any("Dragon scimitar", item(DRAGON_SCIMITAR), item(DRAGON_SCIMITAR_OR))),
```

**Pull request merged**